org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -601,6 +610,39 @@ SsoApplicationManager:
         ]
       }
 
+# Role for a user that can access BedRock in the Synapse LLM prod account
+SSoSynapseLlmDeveloper:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.7/templates/SSO/aws-sso.njk
+  StackName: !Sub '${resourcePrefix}-${appName}-synapsellmdeveloper'
+  StackDescription: 'Role used by LLM Developer group within Synapse LLM prod account'
+  TerminationProtection: false
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: false
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref SynapseLlmProdAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref SynapseLlmProdDeveloperGroup
+    permissionSetName: 'llm-developer'
```

Comment:
this does not match the below, `permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-synapsellmdeveloper-permission-set-arn' ]`

Reply:
My understanding is that at https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/1233/files#diff-c89df69681d7714e12cc8907e3d7e5b10a9bee477bbff86cd269b81ad0c7a1b5R1038, we reference the export name of aws-sso.njk, which is using the stack name. Did I misunderstand?
`Outputs:
  PermissionSetArn:
    Value: !If [ includePermissionSet, !GetAtt PermissionSet.PermissionSetArn, !Ref permissionSetArn ]
    Export:
      Name: !Sub ${AWS::StackName}-permission-set-arn
`

Reply:
sorry, you are correct.

Reply:
Not at all, your answer to my question on Slack prompted me to look ;-)